inductiva/tasks/task.py

```diff
@@ -797,9 +797,12 @@ def _request_download_input_url(
         self
     ) -> Optional[get_tasks_task_id_download_input_url.
                   SchemaFor200ResponseBodyApplicationJson]:
-        api_response = self._api.get_input_download_url(
-            path_params=self._get_path_params(),)
-        return api_response.body
+        try:
+            api_response = self._api.get_input_download_url(
+                path_params=self._get_path_params(),)
+            return api_response.body
+        except exceptions.ApiException:
+            return None
```

Comment:
I'm still not sure if we need to try except here

Reply:
> > Do we wan't to do this? We are just makin an exception vanish without the users knowing.
> > You can just let the exception happen and the client will catch the api exception and print the detail in a nice way. (check log.py file)
> 
> Check how the method `_request_download_output_url` is implemented [here](https://github.com/inductiva/inductiva/blob/main/inductiva/tasks/task.py#L766) — it is similar to the one we are discussing.
> 
> Note that the method `_request_download_input_url` is never called from wait, unlike the method `_request_download_output_url`, where this might happen. In other words, `_request_download_input_url` always follows the path where it returns None, unlike `_request_download_output_url`.
> 
> However, it might be better to print the error message (if present) in this case. Should I add printing the error message (if present) and then return None?
> 
> Thanks for reminding me about the CLI method.

To maintain consistency, we can print a message and return `None`, similar to the other method.

However, please keep this in mind for the future: we already handle `ApiExceptions`. If you don't catch the exception here, it will be caught in the `log.py` file, where a detailed message will be logged automatically (as long as it's an `ApiException`).

We have a process in place for uncaught exceptions, which logs a detailed message about the error.

Reply:
> I'm still not sure if we need to try except here

I think we need -- otherwise the method would never return None.

Reply:
> However, please keep this in mind for the future: we already handle `ApiExceptions`. If you don't catch the exception here, it will be caught in the `log.py` file, where a detailed message will be logged automatically (as long as it's an `ApiException`).

Thanks for the suggestion and explanation! I'll keep that in mind.

Reply:
> I'm still not sure if we need to try except here

You're right. There's no scenario where we would want to return ```None``` in this case -- unlike the similar method ```_request_download_output_url```.